src/dao/LOCsManagerDAO.js

```diff
@@ -34,7 +35,7 @@ class LOCsManagerDAO extends AbstractContractDAO {
         let settingIndex = Setting.get(settingName)
         promises.push(loc.getString(settingName, account).then(r => {
           if (r === value) return
-          return deployed.setLOCString(data.address, settingIndex, this._toBytes32(value), {from: account})
+          return deployed.setLOCString(data.address, settingIndex, bytes32(value), {from: account})
```

Comment:
Why do you use utils/bytes32 instead of AbstractContractDAO._toBytes32? They should do the same
I _toBytes32 is not working properly for you, then we should fix it
Please consider completely removing of the utils/bytes32